common/src/main/resources/assets/gtceu/compass/pages/en_us/circuits/good_electronic_circuit.xml

```diff
@@ -1,26 +1,16 @@
 <page>
- <h1>
-  Good Electronic Circuit
- </h1>
- <h3>
-  Source
- </h3>
- <text><style color="0xaaaaaa" italic="true">Note: this page is ported from MCMOD's GTCEu wiki.<br/>Author: <style color="0xa4e7ff" hover-info="url" underlined="true" url-link="https://center.mcmod.cn/508290/">morningglory</style><br/></style></text>
- <h2 font-color="0xff0000">
-  WARNING!!!
- </h2>
- <text isCenter="true">
-  <style color="0xff0000">The warning means this is not a reliable page.</style>
-  <br/>
-  <style bold="true" color="0xffaa00">Please do not report any mistakes on the page, either discord or github.</style>
-  <br/>
-  This page is translated from the Chinese wiki which hasn't been reviewed and polished yet. It may contain errors and inaccuracies. We are looking forward to your help by improving this page.
- </text>
- <text>
-  <br/>
-  <style>
-   Your second circuit board is also the first generation MV circuit board.
-  </style>
-  <br/>
- </text>
+  <h1>Good Electronic Circuit</h1>
+  <h3>Source</h3>
+  <text>
+    <style color="0xaaaaaa" italic="true">Note: this page is ported from MCMOD's GTCEu wiki.<br></br>Author: <style color="0xa4e7ff" hover-info="url" underlined="true" url-link="https://center.mcmod.cn/508290/">morningglory</style><br></br>, <style color="0xa4e7ff" hover-info="url" underlined="true" url-link="https://github.com/blackdragon2447">BlackDragon2447</style><br></br></style>
+  </text>
+  <text>
+    <br></br>
+    <style>This is your second circuit and also the first generation MV circuit board.</style>
+    <br></br>
+  </text>
+  <h2>Crafting</h2>
+  <recipe id"gtceu:shaped/electronic_circuit_mv"></recipe>
```

Comment:
id="gtceu:shaped/electronic_circuit_mv"

Reply:
Oops, well pr is already merged, you can submit a new pr to fix it, otherwise I'll fix in next pr.